src/lib/cache_lib/intf.ml

```diff
@@ -0,0 +1,111 @@
+open Async_kernel
+open Core_kernel
+
+module Constant = struct
```

Comment:
can you put doccomments on these modules (Constant, Cached, Cache, Transmuter, etc)